.github/dependabot.yml

```diff
@@ -17,3 +17,9 @@ updates:
   # Ignore all patch updates.
   - dependency-name: '*'
     update-types: ["version-update:semver-patch"]
+
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    # Check for updates to GitHub Actions every week
```

Comment:
Month or week?

Reply:
Lemme just strike the whole comment – effect of the configuration is evident enough.

…pushing a `--fixup` commit – TBD if/how Github works with this and `rebase --autosquash`.